web/index.html

```diff
@@ -19,6 +19,7 @@
         <div class="row">
             <div class="col-md-3 col-md-offset-3 mx-auto" style="text-align: center; padding-top: 10px">
                 <h2>Pantherbotics Sign-in</h2>
+                <a href="./records.html">Attendance Records</a>
```

Comment:
Not sure how (very hacky) HTTP server implementation will handle explicit paths like this. I recommend using relative paths, such as href="records.html".

Reply:
Upon further testing this does work as expected, so you are OK as far as paths go. Path relativity seems to be handled by the browser, not the server